components/Workspaces/WorkspaceWelcomeModal.tsx

```diff
@@ -81,7 +81,7 @@ export default function WorkspaceWelcomeModal({ isOpen, onClose }: WorkspaceWelc
     </Drawer>
   ) : (
     <Dialog open={isOpen}>
-      <DialogContent autoStyle={false} onEscapeKeyDown={onClose} onPointerDownOutside={onClose}>
+      <DialogContent aria-label={TITLE} autoStyle={false} onEscapeKeyDown={onClose} onPointerDownOutside={onClose}>
```

Comment:
makes the dialog more accessible and also helps with E2E tests for locating the dialog.